src/Image/Image.php

```diff
@@ -289,14 +288,14 @@ public function setOpacity(float $opacity): self
     /**
      * Rotates an image to $degree degree
      *
-     * @param  int  $degree: The amount to rotate in degrees
+     * @param  ?int  $degree: The amount to rotate in degrees
      * @return Image $image: The rotated image
      *
      * @throws \ImagickException
      */
-    public function setRotation(int $degree): self
+    public function setRotation(?int $degree = null): self
```

Comment:
![Screenshot 2023-04-07 at 2 43 04 AM](https://user-images.githubusercontent.com/20852629/230507088-d30a5d98-bae4-4946-b9a5-f71fbc4dcce5.png)

Reply:
@christyjacob4, what's the point of allowing `null` input?

Reply:
It was to circumvent the error. Looking at it again, it can be re-written as 

```php
public function setRotation(int $degree): self
{
    if ($degree == 0) { return $this; }
}
```

Reply:
The empty() check was unnecessary...

Reply:
I have updated the code 👍